RUFAS/routines/feed_storage/storage.py

```diff
@@ -181,7 +181,7 @@ def process_degradations(self, weather: Weather, time: Time) -> None:
             )
 
             crop.last_time_degraded = copy.deepcopy(time)
-            self.reset_mass_attributes_after_loss(crop, gaseous_dry_matter_loss)
+            self.reset_mass_attributes_after_loss(crop, gaseous_dry_matter_loss, 0.0)
```

Comment:
Maybe add this as a keyword arg
```suggestion
            self.reset_mass_attributes_after_loss(crop, gaseous_dry_matter_loss, moisture_loss=0.0)
```

Reply:
Good suggestion